### PR TITLE
add a menu and a link to Simplified Chinese version document

### PR DESCRIPTION
**Pull request by wu-sheng**
- As I mentioned before in #149, I am translating OpenTracing Documents to Simplified Chinese version . 

- I add a menu, `Other Language Documents`, to choose other languages, if there are other users to contribute translations. As default, I add a link to Simplified Chinese version document.

### CODE REVIEW

File: _docs/SUMMARY.md [hunk 1]
```diff
@@ -10,3 +10,4 @@
   * [Instrumenting Frameworks](pages/instrumentation/instrumenting-frameworks.md)
 * [Authors and Contributors](pages/authors-and-contributors.md)
 * [Supported Tracers](pages/supported-tracers.md)
+* [Other Language Documents](pages/other-language.md)
```
Comment (bensigelman) [inline _docs/SUMMARY.md]: How about `[🇺🇸🇨🇳](pages/translations.md)`?

**Comment by bensigelman** [comment]
@wu-sheng sorry for the delay here, I had a crazy week.

I was looking at what other projects have done with this sort of thing and I couldn't find any examples.

My concern is only that the Simplified Chinese version of the docs will fall out of date. Maybe you could add something to the header of the translated version that specifies the most recent update?

Thanks.

**Comment by wu-sheng** [comment]
@bensigelman , I changed the to `Translations`, and only add `Flag of China`. Because Enlish is the original version, only Chinese version is translations. 

At the same time, I can't find any release version of opentracing.io repository ( maybe consider to add later? ), and I have to choose to add a timestamp at the end of line.

Further, in order to keep translators like me, or any interested people, I commend to add release version, and release note.

**Comment by wu-sheng** [comment]
@bensigelman , are you all in KubeCon?

**Comment by wu-sheng** [comment]
@bensigelman, @yurishkuro  
Is the `translation` link page good enough? or any other opinions?

**Comment by wu-sheng** [comment]
@bensigelman ,  Simplified Chinese version of OT spec released. I am ready to merge to official online doc.

**Comment by bensigelman** [comment]
@wu-sheng sorry for the delay -- can you post a screenshot of how the new link looks in a browser? Thanks.

**Comment by bensigelman** [comment]
Actually, forget it -- just checked it out locally. LGTM, thanks!

**Comment by wu-sheng** [comment]
@bensigelman , I will have a speak in my company, to introduce OT. And I have PowerPoint Document in English, can you give me a e-mail, and help me review it?

**Comment by pritianka** [comment]
I can help you with that if you like! I'm at priyanka@lightstep.com.

Best,
Priyanka

**Comment by wu-sheng** [comment]
@pritianka, Thanks a lot. I will send to you as soon as possible. I am writing it, it will cost more time.

Background, it's the first class of OT spec, and i am in one of top level APM company in China.

**Comment by wu-sheng** [comment]
@pritianka , I have send you a mail from wu.sheng.841108@gmail.com and wu.sheng@foxmail.com, including a powerpoint doc in attachment.

After your review, please let me know on github. Thank you.

**Comment by pritianka** [comment]
Hi! I got your email. I'll send you a response by tomorrow morning if that's cool (it's Tuesday morning in SF right now :-)).

**Comment by wu-sheng** [comment]
@pritianka thank you.

**Comment by pritianka** [comment]
Sent you a note with suggestions!
ᐧ

Priyanka Sharma
Advisor, HeavyBit <http://heavybit.com/>
Head of Marketing & Partnerships, LightStep <http://lightstep.com/>
Co-founder, WakaTime <http://www.wakatime.com/> - Quantify your coding
www.twitter.com/pritianka
650-796-7125

**Comment by bensigelman** [comment]
@wu-sheng is this ready to be merged yet? (i.e., is the destination URL in acceptable condition?)

**Comment by wu-sheng** [comment]
@bensigelman , I got confused about **GitBook** published page. Can you help me, to confirm for me?

My gitbook homepage https://www.gitbook.com/book/wu-sheng/opentracing-io/details is always working. But my published page is often fail down in China: https://wu-sheng.gitbooks.io/opentracing-io/content/

Can you confirm?

**Comment by bensigelman** [comment]
@wu-sheng sure :)

image

image


Does that help?

**Comment by wu-sheng** [comment]
@bensigelman, Ohno....,  it's confirmed the url is right of this rp, disgusting China gov firewall.... 

Maybe, should I add a download link to the translation page? It's always working in China.

**Comment by bensigelman** [comment]
@wu-sheng you would have a better sense of what works for developers in China... I really don't feel qualified to have an opinion.

PS: How disappointing that the government's firewall is so poorly implemented. :)  A translated OpenTracing specification has no bearing on 🇨🇳 politics!! Then again, maybe Trump will be censoring the internet in the 🇺🇸 soon, too :-/

**Comment by wu-sheng** [comment]
@bensigelman , It's a disaster for so many China developers. I will send a rp to add a link to download.

**Pull request merged**